doc/releases/release-notes-3.5.rst

```diff
@@ -100,14 +100,132 @@ Bluetooth
 
 * Audio
 
+  * Fixed bad size of metadata_backup in ASCS
+  * Fixed TMAS characteristic permissions
+  * Fixed BAP bt_bap_scan_delegator_find_state implementation
+  * Fixed possible ASCS ASE stuck in releasing state
+  * Fixed ASCS to drop ISO PDUs if ASE is not in streaming state
+  * Fixed audio stack to accept empty CCID list in audio metadata
+  * Renamed bt_codec to bt_audio_codec_{cap, conf, data}
+  * Fixed OTS bt_ots_init parameter struct naming
```

Comment:
This is not Audio related

Reply:
Github isn't great, but it is specifially this line

```
Fixed OTS bt_ots_init parameter struct naming
```

That isn't audio related

Reply:
Moved